helm/charts/os2iot-frontend/templates/deployment.yaml

```diff
@@ -39,9 +39,7 @@ spec:
             requests:
               cpu: 5m
               memory: 128Mi
-          env:
-            - name: DATABASE_HOSTNAME
-              value: os2iot-postgres-svc
-            - name: DATABASE_PORT
-              value: '5432'
+          envFrom:
```

Comment:
I like this method - but maybe change the envFrom -> envFromConfigMap or similar.  (if it can't just be 'env')

Reply:
Vi can't change it. envFrom is a kubernetes feature and part of the yaml schema. We have to use envFrom in order to use configMapRef